eng/config/PublishData.json

```diff
@@ -51,7 +51,7 @@
         "Shipping",
         "NonShipping"
       ],
-      "vsBranch": "rel/d17.9",
+      "vsBranch": "main",
```

Comment:
This means we have two branches doing insertions into `main` right now. Do we just abandon anything with `P3` on it?

If so... maybe update the title prefix with something like `[DO NOT MERGE]` to make it easy :)

Reply:
We keep creating and monitoring main insertions so we can catch any problems with it before VS snaps.

Reply:
Thanks for the prefix suggestion, I'm going to merge as-is for expediency, but I will include it when I update the docs for the next time we do this.